lib/runBlock.js

```diff
@@ -83,7 +83,7 @@ module.exports = function (opts, cb) {
     async.eachSeries(block.transactions, processTx, cb)
 
     function processTx (tx, cb) {
-      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).cmp(new BN(tx.gasLimit).add(gasUsed)) === -1
+      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).lt(new BN(tx.gasLimit).add(gasUsed))
```

Comment:
Ah sorry, ``true`` to equal if already switched to the equivalent ``gte`` expression:

```
(new BN(tx.gasLimit).add(gasUsed)).gte(new BN(block.header.gasLimit))
```

Which is not the same semantics as ``gasLimitIsHigherThanBlock`` (equivalent would be ``gasLimitIsHigherOrEqualThanBlock``.